src/features/smSystem/suppliersOrders/types/common.ts

```diff
@@ -0,0 +1,11 @@
+export interface IdName {
+  id: number;
+  name: string;
+}
```

Comment:
Chyba dla usprawnienia czytelności + dla przyszłości, nie bałbym się tu o duplikację kodu, zrobiłbym interface Branch i Supplier, bez problemu 
... A widzę w pliku poniżej są takie definicje, to ich bym używał w takim razie i usunął to, bo to dopiero duplikacja xd